start.html

```diff
@@ -99,7 +99,49 @@
 how to find elements on the page. WebDriver offers a number of built-in selector
 types, amongst them finding an element by its ID attribute:
 
-<pre><code class=java>WebElement cheese = driver.findElement(By.id("cheese"));</code></pre>
+    <div class="tabs">
+        <ul class="tabs">
+            <li><a href="#java">java</a></li>
+            <li><a href="#ruby">ruby</a></li>
+            <li><a href="#python">python</a></li>
+            <li><a href="#c-sharp">c#</a></li>
+            <li><a href="#perl">perl</a></li>
+            <li><a href="#php">php</a></li>
+            <li><a href="#jscript">javascript</a></li>
+        </ul>
+
+        <div class="tab" id="ruby">
+            <pre><code class=java>element = driver.find_element(:id, 'cheese')</code></pre>
+        </div>
```

Comment:
Here again drop the outer `<div>`.  We can identify the `<pre>` block to show by finding the immediate descendant from the language list/“tabs” above.